pkg/zkcertificate/certificate_twitter.go

```diff
@@ -0,0 +1,128 @@
+// Copyright © 2024 Galactica Network
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+package zkcertificate
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"strings"
+	"time"
+
+	"github.com/iden3/go-iden3-crypto/poseidon"
+
+	"github.com/galactica-corp/guardians-sdk/internal/validation"
+)
+
+// TwitterInputs represents the input data for X/Twitter verification.
+type TwitterInputs struct {
+	CreatedAt      time.Time `json:"createdAt" validate:"required,datetime=2006-01-02T15:04:05Z07:00,lt"`
```

Comment:
Don't look at the previous example in the docs, it is a completely made up sketch. I propose using this data, because we looked into X/Twitter API and picked what we can use.

Twitter API doesn't provide `likesCount` at all. Let's use exactly this data shape